hs/src/Reach/Connector/ALGO.hs

```diff
@@ -112,7 +112,9 @@ conName' :: T.Text
 conName' = "ALGO"
 
 conCons' :: DLConstant -> DLLiteral
-conCons' DLC_UInt_max = DLL_Int sb $ 2 ^ (64 :: Integer) - 1
+conCons' = \case
+  DLC_UInt_max  -> DLL_Int sb $ 2 ^ (64 :: Integer) - 1
+  DLC_Zero_addr -> DLL_Int sb $ 0
```

Comment:
This is not really correct... Right now the code makes the assumption that "constants can be turned into literals", but this conversion would be of the wrong type, because `Token` is a different type than `Int`